src/main/scala/com/microsoft/hyperspace/index/IndexLogEntry.scala

```diff
@@ -29,7 +31,14 @@ case class NoOpFingerprint() {
 // IndexLogEntry-specific Content that uses IndexLogEntry-specific fingerprint.
 case class Content(root: String, directories: Seq[Content.Directory])
 object Content {
-  case class Directory(path: String, files: Seq[String], fingerprint: NoOpFingerprint)
+  case class Directory(path: String, files: Seq[FileInfo], fingerprint: NoOpFingerprint)
```

Comment:
This will be a breaking change right? We cannot consume v0.2.0 index?

Reply:
yes this will be breaking, it won't consume v020 indexes

Reply:
Can you update "Does this PR introduce any user-facing change?"?

Reply:
done, thanks @imback82